cdap-cli-tests/src/test/java/co/cask/cdap/cli/CLIMainTest.java

```diff
@@ -350,6 +351,16 @@ public void testDataset() throws Exception {
     testCommandOutputContains(cli, "list dataset instances", FakeDataset.class.getSimpleName());
     testCommandOutputContains(cli, "get dataset instance properties " + datasetName, "\"a\":\"1\"");
 
+    // test dataset creation with owner
+    String commandOutput = getCommandOutput(cli, "create dataset instance " + datasetType.getName() + " " +
+      ownedDatasetName + " \"a=1\"" + " " + "someDescription " + ArgumentName.PRINCIPAL +
+      " alice/somehost.net@somekdc.net");
```

Comment:
what happens if I try: create dataset instance table mytable principal alice/somehost.net@somekdc.net
Does it say, error?